share/util/fields.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from cryptography.exceptions import InvalidTag
+from django.db import models
+from django.conf import settings
+import jwe
+
+SENSITIVE_DATA_KEY = jwe.kdf(settings.SECRET_KEY.encode('utf-8'), settings.SALT.encode('utf-8'))
+
+
+class EncryptedJSONField(models.BinaryField):
+    '''
+    This field transparently encrypts data in the database. It should probably only be used with PG unless
+    the user takes into account the db specific trade-offs with TextFields.
+    '''
+    prefix = 'jwe:::'
```

Comment:
If you declare `prefix` as bytes (`b'jwe:::'`) you won't have to call `.encode(` on it.